contracts/GaugeController.sol

```diff
@@ -99,28 +199,63 @@ contract GaugeController is IGaugeController {
         gaugeScaleTwab.details = twabDetails;
     }
 
+    /**
+     * @notice Read Gauge balance.
+     * @param _gauge Address of existing Gauge
+     * @return uint256 GaugeTWAB.details.balance
+    */
     function getGauge(address _gauge) public view returns (uint256) {
         return gaugeTwabs[_gauge].details.balance;
     }
 
+    /**
+     * @notice Read Gauge scaled balance.
+     * @param _gauge Address of existing Gauge
+     * @return uint256 GaugeScaleTWAB.details.balance
+    */
     function getGaugeScale(address _gauge) public view returns (uint256) {
         return gaugeScaleTwabs[_gauge].details.balance;
     }
 
+    /**
+     * @notice Calculate Gauge weighted balance using Staked AND Scaled time-weighted average balances.
+     * @param _gauge Address of existing Gauge
+     * @param _startTime Unix timestamp to signal START of the Binary search
+     * @param _endTime Unix timestamp to signal END of the Binary search
+     * @return uint256 Weighted(Staked * Scaled) Gauage Balance
+    */
     function getScaledAverageGaugeBetween(address _gauge, uint256 _startTime, uint256 _endTime) external override view returns (uint256) {
```

Comment:
Should be named `getScaledAverageGaugeBalanceBetween`, it's pretty confusing otherwise with the similar `getAverageGaugeScaleBetween`.
```suggestion
    function getScaledAverageGaugeBalanceBetween(address _gauge, uint256 _startTime, uint256 _endTime) external override view returns (uint256) {
```

Reply:
Committed